components/Search/Hit.tsx

```diff
@@ -77,22 +93,80 @@ If neither the excerpt or the description is a match (like the match can be for
   a resonsible person) we will fallback to either the excerpt or description, like
   in the static text without any query)
 */
+
 const getSnippetAttribute = (hit: HitProps["hit"]) => {
   // eslint-disable-next-line no-underscore-dangle
   const excerptSnippet = hit._snippetResult?.excerpt as HitAttributeSnippetResult
   // eslint-disable-next-line no-underscore-dangle
   const descriptionSnippet = hit._snippetResult?.description as HitAttributeSnippetResult
+
   if (hit.excerpt && excerptSnippet && excerptSnippet.matchLevel === ("full" || "partial")) {
     return "excerpt"
     /* eslint no-else-return: "error"  */
   } else if (hit.description && descriptionSnippet && descriptionSnippet.matchLevel === ("full" || "partial")) {
     return "description"
   }
+
   return hit.excerpt ? "excerpt" : "description"
 }
 
+const isNewDataProduct = (hit: HitProps["hit"]) => {
+  const currentDate = new Date()
+  const compareDate = new Date(currentDate.setMonth(currentDate.getMonth() - 1))
+  const createdAtDate = new Date(hit.createdAt)
+  return createdAtDate.getTime() >= compareDate.getTime()
+}
+
 export const Hit = ({ hit, hasQuery }: HitProps) => {
   const { id, community = [], tags = [] } = hit
+  const intl = useIntl()
+
+  if (isNewDataProduct(hit)) {
+    return (
+      <StyledLink href={{ pathname: "/assets/[id]", query: { id } }}>
+        <TopContainer>
+          {community &&
+            community.map((item) => (
+              <Typography variant="overline" style={{ fontSize: "0.75rem", fontWeight: "medium" }} key={item}>
+                {item}
+              </Typography>
+            ))}
+          <StyledChipNew style={{ fontSize: "0.75rem", fontWeight: "medium" }}>
+            {intl.formatMessage({ id: "search.results.new.dataproduct" })}
+          </StyledChipNew>
+        </TopContainer>
+        <Heading size="lg" level="h2" bold style={{ marginBottom: tokens.spacings.comfortable.medium }}>
+          <StyledName
+            hit={hit}
+            attribute="name"
+            classNames={{
+              highlighted: "highlighted",
+            }}
+          />
+        </Heading>
+        {hasQuery ? (
+          <StyledTypography variant="body_short">
+            <StyledSnippet
+              hit={hit}
+              attribute={getSnippetAttribute(hit)}
+              classNames={{
+                highlighted: "highlighted",
+              }}
+            />
+          </StyledTypography>
+        ) : (
+          <TruncatedStaticExcerpt variant="body_short">{hit.excerpt || hit.description}</TruncatedStaticExcerpt>
+        )}
+        {tags.length > 0 && (
+          <TagsContainer>
+            {tags.map((item) => (
+              <StyledChip key={item}>{item}</StyledChip>
+            ))}
+          </TagsContainer>
+        )}
+      </StyledLink>
+    )
+  }
```

Comment:
i would strongly suggest, instead of rendering the entire component with a new tag if it's a new data product, that you render the new tag if it's a new data product. the difference being:
```js
// instead of
if (isNew()) {
  // entire component
}
// we do
return (
  <StyledLink>
    <FlexContainer>
      {community && <Community />}
      {isNew() && <NewTag />}
    </FlexContainer>
  </StyledLink>
)
```
doing this means that if we need to make a change to the hit card, we don't need to do it two places :+1: